packages/blockchain-link-utils/src/tests/fixtures/solana.ts

```diff
@@ -89,47 +85,41 @@ const parsedTransactions = {
     basic: {
         transaction: {
             meta: {
-                computeUnitsConsumed: 100,
-                preBalances: [200, 100],
-                postBalances: [180, 110],
-                fee: 10,
+                computeUnitsConsumed: 100n,
+                preBalances: [200n, 100n],
+                postBalances: [180n, 110n],
+                fee: 10n,
```

Comment:
By and large, the new API spits off the _correct_ number types – `bigint` when the corresponding type on the server is `u64` and `number` otherwise.